src/components/map-container.tsx

```diff
@@ -43,35 +42,34 @@ import {
   updateMapboxLayers,
   Layer,
   LayerBaseConfig,
-  VisualChannelDomain
-} from '@kepler.gl/layers';
-import {setLayerBlending} from 'utils/gl-utils';
-import {transformRequest} from 'utils/map-style-utils/mapbox-utils';
-import {
   getLayerHoverProp,
   renderDeckGlLayer,
   prepareLayersToRender,
   prepareLayersForDeck,
-  LayerHoverProp
-} from 'utils/layer-utils';
+  LayerHoverProp,
+  VisualChannelDomain
+} from '@kepler.gl/layers';
+import {Filter, InteractionConfig, MapState, SplitMapLayers} from '@kepler.gl/types';
+import {
+  errorNotification,
+  setLayerBlending,
+  transformRequest,
+  observeDimensions,
+  unobserveDimensions
+} from '@kepler.gl/utils';
 
 // default-settings
 import {ThreeDBuildingLayer} from '@kepler.gl/deckgl-layers';
 import {FILTER_TYPES, GEOCODER_LAYER_ID, THROTTLE_NOTIFICATION_TIME} from '@kepler.gl/constants';
 
 import ErrorBoundary from 'components/common/error-boundary';
-import {observeDimensions, unobserveDimensions} from '../utils/observe-dimensions';
 import {LOCALE_CODES} from '@kepler.gl/localization';
 import {
-  Datasets,
-  Filter,
-  InteractionConfig,
   MapControls,
-  MapState,
   MapStyle,
   Viewport
 } from 'reducers';
-import {SplitMapLayers} from 'reducers/vis-state-updaters';
+import { Datasets } from 'reducers/table-utils';
```

Comment:
rebase with master should set lint to run on these files, which will catch this spacing issue here